manticore/core/state.py

```diff
@@ -364,3 +364,12 @@ def mem(self):
 
     def _init_context(self):
         self.context['branches'] = dict()
+
+    def generate_testcase(self, name, message='State generated testcase'):
+        """
+        Generate a testcase for this state and place in the analysis workspace.
+
+        :param str name: Short string identifying this testcase used to prefix workspace entries.
+        :param str message: Longer description
+        """
+        self.publish('state_generate_inputs', name, message)
```

Comment:
I'm not sure this fits with our existing event naming scheme of will/did. Maybe `should_generate_inputs` is more appropriate here?